vjudge/submit.py

```diff
@@ -0,0 +1,71 @@
+"""
+The MIT License (MIT)
+
+Copyright (c) 2014 NTHUOJ team
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+"""
+import requests, base64, json, bs4
+from problem.models import Submission, Problem
+from vjudge.models import VjudgeID
+from vjudge.status import status_url
+from utils.config_info import get_config
+from utils.log_info import get_logger
+
+logger = get_logger()
+
+vjudge_username = get_config('vjudge', 'username')
+vjudge_password = get_config('vjudge', 'password')
+login_url = 'http://acm.hust.edu.cn/vjudge/user/login.action'
+submit_url = 'http://acm.hust.edu.cn/vjudge/problem/submit.action'
+
+LANGUAGE_CHOICE = {
+    Problem.UVA_JUDGE: {Problem.C: 1 ,Problem.CPP: 3, Problem.CPP11: 5},
```

Comment:
I have no idea about that. Maybe @bruce3557 can provide some advice.
(I took it to be some magic numbers XD)